### DISCUSSION TITLE
Message to <support@gap-system.org> was undeliverable

### DISCUSSION THREAD

**Discussion by mikepierce** [gap-system/gap#4244] [General]
![2021-02-03-220041_1014x149_scrot](https://user-images.githubusercontent.com/10717115/106851555-5182ea00-666b-11eb-88a7-121cb806cbf9.png)

Is this an error or is that email no longer maintained?

**Comment by olexandr-konovalov** [comment]
@mikepierce must be an error, the address is correct. Could you please try again?

**Reply by mikepierce** [reply]
Hey @olexandr-konovalov! I tried again, and again it failed to deliver.

**Reply by olexandr-konovalov** [reply]
Strange... I've just sent a test message and it went through.

**Reply by mikepierce** [reply]
Does the support email not allow attachments? Or txt formatting? Here's the email I was trying to send through:

![2021-02-04-145131_996x649_scrot](https://user-images.githubusercontent.com/10717115/106965490-81c49a00-66f8-11eb-98f8-59c355a3968b.png)

**Reply by olexandr-konovalov** [reply]
It allows txt formatting (which is preferred!) but doesn't allow attachments. Still, I'd expect another response. Would you mind trying again as a plain text with no attachments, please?

**Reply by mikepierce** [reply]
Removing the attachment and all txt formatting resulted in the same error. But I just noticed that the error message looks like it was from my University's mail server `@mx2.ucr.edu`. Looking up [DKIM](http://dkim.org/info/dkim-faq.html), I don't understand it, so I can't tell who's side is error is on. It's not a helpful error. I've forwarded this email to my University's IT to see what they have to say. 

Meanwhile I've sent this from another email and I haven't gotten back an error yet, so I'm gonna conclude this is something with my University's servers.

**Reply by olexandr-konovalov** [reply]
The other email went through! Hopefully someone will be able to help.

**Comment by dimpase** [comment]
A modern way to report issues is to use the GitHub issues/discussions, no? 

Why does https://www.gap-system.org/Contacts/contacts.html not even mention this,  but still hammers on support@
(which often delays messages for many days :-( )?

**Reply by mikepierce** [reply]
I mean, this isn't really an issue with the GAP code that I can tell. So GitHub issues at least seemed like the wrong place to bring it up. And I'm asking for help rather than a discussion, so GitHub discussions doesn't seem like a good fit either. It is a bummer though that I only found out my code wouldn't work after trying to run it, and that I didn't see a mention that my special "*case*"—I'm not even sure what "case" the error message is talking about—wasn't supported in the docs. 

I haven't heard back from support yet. Should I post a new discussion about what I'd like help with?

**Reply by dimpase** [reply]
support@ is a closed mailing list, a small group of very busy with other tasks people, and, perhaps even more important, without an open archive.
An open forum has an advantage that it becomes a body of knowledge searchable for everyone.

On support@ many questions are asked again and again, on the other hand.
It should be limited to topic that should not be made public.


GitHub "discussions" is how GitHub names this facility, not that it must be limited to discussions.

Same with GitHub "issues".

**Reply by mikepierce** [reply]
Okay. I started a [discussion](https://github.com/gap-system/gap/discussions/4247).
